Explorer/Assets/DCL/Navmap/NavmapController.cs

```diff
@@ -33,129 +27,95 @@ public class NavmapController : IMapActivityOwner, ISection, IDisposable
         private const string EMPTY_PARCEL_NAME = "Empty parcel";
         private const string WORLDS_WARNING_MESSAGE = "This is the Genesis City map. If you jump into any of this places you will leave the world you are currently visiting.";
         private const MapLayer ACTIVE_MAP_LAYERS =
-            MapLayer.SatelliteAtlas | MapLayer.PlayerMarker | MapLayer.ParcelHoverHighlight | MapLayer.ScenesOfInterest | MapLayer.Favorites | MapLayer.HotUsersMarkers | MapLayer.Pins;
+            MapLayer.SatelliteAtlas | MapLayer.ParcelsAtlas | MapLayer.PlayerMarker | MapLayer.ParcelHoverHighlight | MapLayer.ScenesOfInterest | MapLayer.Favorites | MapLayer.HotUsersMarkers | MapLayer.Pins | MapLayer.SearchResults | MapLayer.LiveEvents |
+            MapLayer.Category;
 
         private readonly NavmapView navmapView;
         private readonly IMapRenderer mapRenderer;
         private readonly NavmapZoomController zoomController;
-        private readonly NavmapFilterController filterController;
         private readonly NavmapSearchBarController searchBarController;
         private readonly RectTransform rectTransform;
         private readonly SatelliteController satelliteController;
+        private readonly PlaceInfoToastController placeToastController;
+        private readonly IPlacesAPIService placesAPIService;
         private readonly IRealmData realmData;
         private readonly IMapPathEventBus mapPathEventBus;
-        private readonly SectionSelectorController<NavmapSections> sectionSelectorController;
-        private readonly Dictionary<NavmapSections, TabSelectorView> tabsBySections;
-        private readonly Dictionary<NavmapSections, ISection> mapSections;
-        private readonly Mouse mouse;
+        private readonly UIAudioEventsBus audioEventsBus;
+        private readonly PlacesAndEventsPanelController placesAndEventsPanelController;
         private readonly StringBuilder parcelTitleStringBuilder = new ();
         private readonly NavmapLocationController navmapLocationController;
+        private CancellationTokenSource? fetchPlaceAndShowCancellationToken = new ();
 
-        private CancellationTokenSource animationCts;
-        private IMapCameraController cameraController;
+        private CancellationTokenSource? animationCts;
+        private IMapCameraController? cameraController;
 
         private Vector2 lastParcelHovered;
         private NavmapSections lastShownSection;
         private MapRenderImage.ParcelClickData lastParcelClicked;
 
         public IReadOnlyDictionary<MapLayer, IMapLayerParameter> LayersParameters { get; } = new Dictionary<MapLayer, IMapLayerParameter>
             { { MapLayer.PlayerMarker, new PlayerMarkerParameter { BackgroundIsActive = true } } };
-        public FloatingPanelController FloatingPanelController { get; }
+
+        // TODO: we need this property for the MVCManagerAnalyticsDecorator only.. something is not right in the design

```

Comment:
Okay, but you are using it nonetheless in this class. Is this comment relevant?

Reply:
for @lorux0

Reply:
Deprecated. Something changed at MVManagerAnalyticsDecorator and we dont need to expose it anymore